### PR TITLE
Update imlakshay08.json

### PR DESCRIPTION
**Pull request by imlakshay08**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1689"></a>

### CODE REVIEW

**Review by imlakshay08 — COMMENTED** [review]
All checks passed, please merge it.

File: public/data/imlakshay08.json [hunk 1]
```diff
@@ -13,6 +13,16 @@
...
       "url": "https://twitter.com/imLakshay08",
       "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/imlakshay08/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Mail me on",
```
Comment (Panquesito7) [inline public/data/imlakshay08.json:23]: What do you think about this?
```suggestion
      "name": "Contact me via mail",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (Pradumnasaraf) [reply public/data/imlakshay08.json:23]: I think it's a personal preference @Panquesito7, so it's good!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — APPROVED** [review]
Good to go! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**